src/builder/builder.js

```diff
@@ -62,10 +62,7 @@ module.exports = function (createChunker, ipldResolver, createReducer, _options)
     waterfall([
       (cb) => DAGNode.create(d.marshal(), cb),
       (node, cb) => {
-        ipldResolver.put({
-          node: node,
-          cid: new CID(node.multihash)
-        }, (err) => cb(err, node))
+        ipldResolver.put(node, new CID(node.multihash), (err) => cb(err, node))
```

Comment:
this is much worse than before in terms of readability (the change in the resolver api)